pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -63,44 +70,48 @@ def load_earth_free_air_anomaly(
     registration
         Grid registration type. Either ``"pixel"`` for pixel registration or
         ``"gridline"`` for gridline registration. Default is ``None``, means
-        ``"gridline"`` for all resolutions except ``"01m"`` which is
-        ``"pixel"`` only.
+        ``"gridline"`` for all resolutions except ``"01m"`` which is ``"pixel"`` only.
+    uncertainty
+        Set to ``True`` to download the uncertainties related to the free-air anomaly
+        values [Default is ``False`` and gives the altimetry-based marine free-air
+        anomaly values.
```

Comment:
Maybe better?
```suggestion
    uncertainty
        By default, the Earth free-air anomaly values are returned. Set to ``True`` to 
        return the the uncertainties instead.
```

Reply:
This update is done in https://github.com/GenericMappingTools/pygmt/pull/3727/commits/4665c61c06a68aaadeb9c46defab3e9bc6940bba.